x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -106,104 +104,109 @@ export class SpaceSelector extends Component<Props, State> {
-                />
-              </p>
-            </EuiText>
-          </EuiPageHeader>
-          <EuiPageContent className="spcSpaceSelector__pageContent">
-            <EuiFlexGroup
-              // @ts-ignore
-              direction="column"
-              alignItems="center"
-              responsive={false}
-            >
-              {this.getSearchField()}
-            </EuiFlexGroup>
-
-            <EuiSpacer size="xl" />
-
-            {this.state.loading && <EuiLoadingSpinner size="xl" />}
-
-            {!this.state.loading && (
-              <SpaceCards spaces={filteredSpaces} serverBasePath={this.props.serverBasePath} />
-            )}
-
-            {!this.state.loading && !this.state.error && filteredSpaces.length === 0 && (
-              <Fragment>
-                <EuiSpacer />
-                <EuiText color="subdued" textAlign="center">
+      <KibanaPageTemplate
+        template="empty"
+        className="spcSpaceSelector"
+        data-test-subj="kibanaSpaceSelector"
+        pageContentBodyProps={{ className: 'spcSpaceSelector__pageContent' }}
+      >
+        <EuiText textAlign="center" size="s">
+          <EuiSpacer size="xxl" />
+          <KibanaSolutionAvatar name="Elastic" size="xl" />
+          <EuiSpacer size="xxl" />
+          <h1
+            // plain `eui` class undos forced focus style on non-EUI components
+            className="eui spcSpaceSelector__pageHeader"
+            tabIndex={0}
+            ref={this.setHeaderRef}
+          >
+            <FormattedMessage
+              id="xpack.spaces.spaceSelector.selectSpacesTitle"
+              defaultMessage="Select your space"
+            />
+          </h1>
+          <EuiTextColor color="subdued">
+            <p>
+              <FormattedMessage
+                id="xpack.spaces.spaceSelector.changeSpaceAnytimeAvailabilityText"
+                defaultMessage="You can change your space at anytime."
+              />
+            </p>
+          </EuiTextColor>
+        </EuiText>
+        <EuiSpacer size="xxl" />
+
+        {this.getSearchField()}
+
+        <EuiSpacer size="xl" />
+
+        {this.state.loading && <EuiLoadingSpinner size="xl" />}
+
+        {!this.state.loading && (
+          <SpaceCards spaces={filteredSpaces} serverBasePath={this.props.serverBasePath} />
+        )}
+
+        {!this.state.loading && !this.state.error && filteredSpaces.length === 0 && (
+          <Fragment>
+            <EuiSpacer />
+            <EuiPanel className="spcSpaceSelector__errorPanel" color="subdued">
+              <EuiTitle size="xs">
+                <h2>
                   <FormattedMessage
                     id="xpack.spaces.spaceSelector.noSpacesMatchSearchCriteriaDescription"
-                    defaultMessage="No spaces match search criteria"
+                    defaultMessage="No spaces match"
+                  />{' '}
+                  {`"${this.state.searchTerm}"`}
```

Comment:
I would put the search term in the message string as a variable to ensure this works for languages other than English: 

```typescript
<FormattedMessage 
  id="xpack.spaces.spaceSelector.noSpacesMatchSearchCriteriaDescription"
  defaultMessage="No spaces match \"{searchTerm}\""
  values={{ searchTerm: this.state.searchTerm }}
/>
```

Reply:
Makes sense, are you sure of this format though? The escaping of the quotes doesn't seem to work.
<img width="544" alt="Screen Shot 2022-03-28 at 11 16 13 AM" src="https://user-images.githubusercontent.com/549577/160430629-9d969cf4-06e7-49f7-a1b1-34afc1d0f26e.png">

Reply:
Sorry I didn't test this. The following should work:

```typescript
<FormattedMessage 
  id="xpack.spaces.spaceSelector.noSpacesMatchSearchCriteriaDescription"
  defaultMessage={'No spaces match "{searchTerm}"'}
  values={{ searchTerm: this.state.searchTerm }}
/>
```

Reply:
I swtiched to using the `i18n.translate()` function instead and it seems to work:

```
{i18n.translate(
 'xpack.spaces.spaceSelector.noSpacesMatchSearchCriteriaDescription',
  {
    defaultMessage: 'No spaces match "{searchTerm}"',
    values: { searchTerm: this.state.searchTerm },
  }
)}
```

Reply:
Haha, sorry, crossed-paths. I can switch back to `FormattedMessage` is preferred, or just leave as `i18n.translate()`.